builtin/providers/google/resource_google_project_test.go

```diff
@@ -214,35 +214,16 @@ resource "google_project" "acceptance" {
 `, pid)
 }
 
-func testAccGoogleProjectImportExistingWithIam(pid string) string {
```

Comment:
While you're deleting uncalled functions, want to also delete `testAccGoogleProjectImportExisting`? (or if you think that should actually be made into a test, maybe add a todo)